mungegithub/submit-queue/deployment/test-infra/configmap.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: test-infra-sq-config
+data:
+  # basic config options.
+  config.http-cache-dir: /cache/httpcache
+  config.organization: kubernetes
+  config.project: test-infra
+  config.pr-mungers: blunderbuss,lgtm-after-commit,submit-queue,check-labels
+  config.state: open
+  config.token-file: /etc/secret-volume/token
+  gitrepo.repo-dir: /gitrepos
+  gitrepo.enable-md-yaml: "false"
+
+  # test-options feature options.
+  test-options.required-retest-contexts: "\"\""
+
+  # submit-queue options.
+  submit-queue.required-contexts: "continuous-integration/travis-ci/pr"
+  submit-queue.nonblocking-jenkins-jobs: "\"\""
+  submit-queue.jenkins-jobs: "\"\""
+  submit-queue.presubmit-jobs: "\"\""
+  submit-queue.weak-stable-jobs: "\"\""
+  submit-queue.do-not-merge-milestones: "\"\""
+  submit-queue.admin-port: "9999"
+  submit-queue.chart-url: ""
+  submit-queue.history-url: ""
+
+  # options used by the GCS feature.
```

Comment:
These are useful only if we use the jenkins instances. They can be blank here since we run only the travis presubmit and no other tests.